java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1285,13 +1309,37 @@ public void sadd_srem_scard_smembers_existing_set(BaseClient client) {
         Set<String> expectedMembers = Set.of("member1", "member2", "member4");
         assertEquals(expectedMembers, client.smembers(key).get());
 
-        Set<GlideString> expectedMembersBin = Set.of(gs("member1"), gs("member2"), gs("member4"));
-        assertEquals(expectedMembersBin, client.smembers(gs(key)).get());
+        Set<String> expectedMembersBin = Set.of("member1", "member2", "member4");
+        assertEquals(expectedMembersBin, client.smembers(key).get());
```

Comment:
why? This duplicates the previous check then. If it was added by mistake - just delete.